studio/utils/checkForDuplicateLocations.ts

```diff
@@ -0,0 +1,21 @@
+interface LocationReference {
```

Comment:
is this a new util or was it not added somewhere by @mathiazom

Reply:
Yes it is, i just needed it for the location validation

Reply:
ah!

Reply:
Have you updated it in bonusesByLocation @mathias? I can do it if not ☺️

Reply:
Yes, I have updated `bonusesByLocation` to use the common util in #535